imgur-screenshot.sh

```diff
@@ -94,6 +105,9 @@ if [ "${1}" = "--check" ]; then
     fi
     (which screencapture &>/dev/null && echo "OK: found screencapture") || echo "ERROR: screencapture not found"
     (which pbcopy &>/dev/null && echo "OK: found pbcopy") || echo "ERROR: pbcopy not found"
+  elif is_gnome; then
```

Comment:
This should go further down to (only) determine if `scrot` or `gnome-screenshot`.

Reply:
You're right because I introduced a bug by missing the xclip call. Thanks.